ci/test_python.sh

```diff
@@ -5,9 +5,9 @@
 
 set -euo pipefail
 
-. /opt/conda/etc/profile.d/conda.sh
+RAPIDS_VERSION_NUMBER=$(rapids-generate-version)
```

Comment:
Other RAPIDS libraries get the current version by running this script

So doing the same thing here